apps/admin_app/assets/js/views/product/edit.js

```diff
@@ -28,28 +28,28 @@ export function handleImageSelect() {
 }
 
 function selectedFile(e) {
-  if(!e.target.files || !window.FileReader) return;
+  if (!e.target.files || !window.FileReader) return;
 
   selDiv.empty()
   var files = e.target.files;
   let filesArr = Array.prototype.slice.call(files);
 
-  filesArr.forEach(function(f) {
-    if(!f.type.match("image.*")) {
+  filesArr.forEach(function (f) {
+    if (!f.type.match("image.*")) {
       return;
     }
     storedFile.push(f)
     var reader = new FileReader();
     reader.onload = function (e) {
-      var html = "<img src=\"" + e.target.result + "\">" + f.name + "<br clear=\"left\"/>";
+      var html = "<div class=\"col-6 px-1 mb-2\"><div class=\"media\"><img class=\"align-self-start mr-3\" src=\"" + e.target.result + "\" alt=\"Generic placeholder image\" /> <div class=\"media-body\"><h5 class=\"mt-0\">" + f.name + "</h5> </div> </div></div>";
```

Comment:
we are using `babel`, so we can use tild sign for multline string that would look cleaner.

Something like

```javascript
var html = `
<div class="col-6 px-1 mb-2">
  <div class=\"media\">
  </div>
</div>";
```

Reply:
I have to work on this section for the image separate tab